include/swift/AST/DiagnosticsSema.def

```diff
@@ -1718,7 +1718,7 @@ ERROR(functions_mutating_and_not,none,
       "method must not be declared both %0 and %1",
       (SelfAccessKind, SelfAccessKind))
 ERROR(static_functions_not_mutating,none,
-      "static functions must not be declared mutating", ())
+      "nonmutating or mutating modifiers cannot be used on static functions because they do not operate on instance properties or self", ())
```

Comment:
Instead of clamping mutating and nonmutating together it would be better to print one or the other depending on what the attribute was, a boolean flag could be used to distinguish between the two (i.e. `no_overloads_match_exactly_in_call` does a similar thing reference vs. call). 

Instead of "operate" we can use "cannot access" and the "because" part can be turned into a note.

Reply:
@xedin 
Thanks for the advice!
I fixed based on your suggestion.